extras/manifests/prometheus-rules.yaml

```diff
@@ -93,6 +98,18 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: CephNodeDown
+      annotations:
+        description: Ceph Storage node {{ $labels.node }} went down. Please check
```

Comment:
Remove the word `Ceph` from description.

Reply:
ack